src/store/items/actions.js

```diff
@@ -66,6 +66,15 @@ export default {
     dispatch('_findAndSetEdittingItemIndex', itemWithTemporaryId);
   },
 
+  _chooseBetweenAddOrUpdateItemOnServer({ dispatch, getters }) {
```

Comment:
the verb is not about doing but just about choosing
it's confusing

also
it could be just "save"